Source/PluginProcessor.cpp

```diff
@@ -169,6 +169,20 @@ void PinkTromboneAudioProcessor::processBlock (AudioBuffer<float>& buffer, MidiB
     // this code if your algorithm always overwrites all the output channels.
     for (auto i = totalNumInputChannels; i < totalNumOutputChannels; ++i)
         buffer.clear (i, 0, buffer.getNumSamples());
+	
+	for (auto meta : midiMessages) {
+		auto currentMessage = meta.getMessage();
+
+		if (currentMessage.isNoteOn()){
+			auto currentNote = currentMessage.getNoteNumber();
+			double midiNoteInHz = juce::MidiMessage::getMidiNoteInHertz(currentNote);
+			this->glottis->frequency = midiNoteInHz;

```

Comment:
I don't love this, honestly. Rather than moving all these frequency parameters to public variables and then setting them all here in PluginProcessor, why not create a new function in glottis, `setFrequency` or something, and then set the individual private member variables in that function? Also, do we know what each of these frequency parameters do? Do we need them all? It seems a little crazy that we're representing frequency in 5 distinct places, so I'm curious if there's a good reason for it.